rules/cross-platform/impact_hosts_file_modified.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/07/07"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+The hosts file on endpoints is used to control manual IP to hostname resolutions. The hosts file is the first point of
+lookup for DNS hostname resolution so if adversaries can modify the endpoint hosts file, they can route traffic to
+malicious infrastructure. This rule detects modifications to the hosts file on Microsoft Windows, Linux (Ubuntu or RHEL)
+and macOS systems. For Windows systems, this rule requires adding "C:/Windows/System32/drivers/etc" as an additional
+path in the "file_integrity" module of auditbeat.yml.
```

Comment:
Can we include a reference to help people modify their `auditbeat.yml` configuration or is there a level of knowledge that we're assuming that users have?

Reply:
By default, `auditbeat.yml` has examples in the in `file_integrity` section. 
```
...
# The file integrity module sends events when files are changed (created,
# updated, deleted). The events contain file metadata and hashes.
- module: file_integrity
  paths:
  - /bin
  - /usr/bin
  - /sbin
  - /usr/sbin
  - /etc
...
```

https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat-reference-yml.html

I'm happy to provide one, but I'm not sure where to put it for reference? Open to suggestions.

Reply:
What do you think about moving the following text from the `description` field to the `notes` field?

```
For Windows systems, this rule requires adding "C:/Windows/System32/drivers/etc" as an additional
path in the "file_integrity" module of auditbeat.yml.
```

Reply:
Got it. 

Added the note and reference fields.